repos.json

```diff
@@ -244,6 +220,102 @@
       "git": "git@gitlab.cee.redhat.com:RHELReporting/telltale-ui.git",
       "name": "Telltale",
       "private": true
+    },
+    {
+      "git": "https://github.com/aerogear/uniforms-patternfly",
+      "name": "Aerogear-Uniforms-Patternfly"
+    },
+    {
+      "git": "https://github.com/ansible-community/ara-web",
+      "name": "Ansible-Community-Ara-Web"
+    },
+    {
+      "git": "https://github.com/openshift/hac-dev",
+      "name": "HAC-Dev"
+    },
+    {
+      "git": "https://github.com/ansible/awx",
+      "name": "Ansible-awx"
+    },
+    {
+      "git": "https://github.com/apache/camel-karavan",
+      "name": "Camel-Karavan"
+    },
+    {
+      "git": "https://github.com/Apicurio/apicurio-registry",
+      "name": "Apicurio-Registry"
+    },
+    {
+      "git": "https://github.com/change-metrics/monocle",
+      "name": "Change-Metrics-Monocle"
+    },
+    {
+      "git": "https://github.com/cockpit-project/cockpit-certificates",
+      "name": "Cockpit-certificates"
+    },
+    {
+      "git": "https://github.com/cockpit-project/cockpit-machines",
+      "name": "Cockpit-machines"
+    },
+    {
+      "git": "https://github.com/cockpit-project/cockpit-podman",
+      "name": "Cockpit-podman"
+    },
+    {
+      "git": "https://github.com/cockpit-project/starter-kit",
+      "name": "Cockpit-Starter-Kit"
+    },
+    {
+      "git": "https://github.com/debezium/debezium-ui",
+      "name": "Debezium"
+    },
+    {
+      "git": "https://github.com/FNNDSC/ChRIS_store_ui",
+      "name": "ChRIS"
+    },
+    {
+      "git": "https://github.com/keycloak/keycloak-quickstarts",
+      "name": "Keycloak-Quickstarts"
+    },
+    {
+      "git": "https://github.com/kiali/kiali",
+      "name": "Kiali"
+    },
+    {
+      "git": "https://github.com/kiegroup/optaweb-employee-rostering",
+      "name": "Optaweb-Employee-Rostering"
+    },
+    {
+      "git": "https://github.com/kiegroup/optaweb-vehicle-routing",
+      "name": "Optaweb-Vehicle-Routing"
+    },
+    {
+      "git": "https://github.com/kubernetes-sigs/kui",
+      "name": "Kui"
+    },
+    {
+      "git": "https://github.com/ocristopfer/cockpit-sensors",
+      "name": "Ocristopher"
+    },
+    {
+      "git": "https://github.com/opendatahub-io/jupyterhub-singleuser-profiles",
+      "name": "Opendatahub"
+    },
+    {
+      "git": "https://github.com/openshift/console-plugin-template",
+      "name": "Console-Plugin-Template"
+    },
+    {
+      "git": "https://github.com/packit/dashboard",
+      "name": "Packit-Dashboard"
+    },
+    {
+      "git": "https://github.com/patternfly/patternfly-react-seed",
```

Comment:
Confirming if we want to track this - though a small project, could this skew any data we'd report on?  Ideally if consumers are utilizing this as a starting point we can track them elsewhere in this data.

Otherwise LGTM 👍

Reply:
Hey @Evan - react seed or the packit dashboard?

Reply:
@janwright73 Patternfly-react-seed